Table.put is failing in samples due to TypeError: 'dict_values' object does not support indexing

Caused once you upgrade to version 0.32.0. The tests also fail at current 0.31.0 but for different reasons.

https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/bigtable/hello_happybase

@tseaver 
============================================================ FAILURES =============================================================
____________________________________________________________ test_main ____________________________________________________________
Traceback (most recent call last):
  File "/Users/thebilly/cloud/python-docs-samples/.nox/py36-sample-bigtable-hello_happybase/lib/python3.6/site-packages/google/cloud/happybase/table.py", line 428, in put
    batch.put(row, data)
  File "/Users/thebilly/cloud/python-docs-samples/.nox/py36-sample-bigtable-hello_happybase/lib/python3.6/site-packages/google/cloud/happybase/batch.py", line 162, in put
    column_pairs = _get_column_pairs(six.iterkeys(data),
  File "/Users/thebilly/cloud/python-docs-samples/.nox/py36-sample-bigtable-hello_happybase/lib/python3.6/site-packages/six.py", line 581, in iterkeys
    return iter(d.keys(**kw))
AttributeError: 'list' object has no attribute 'keys'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/thebilly/cloud/python-docs-samples/bigtable/hello_happybase/main_test.py", line 32, in test_main
    table_name)
  File "/Users/thebilly/cloud/python-docs-samples/bigtable/hello_happybase/main.py", line 75, in main
    table.put(row_key, list({column_name: value}))
  File "/Users/thebilly/cloud/python-docs-samples/.nox/py36-sample-bigtable-hello_happybase/lib/python3.6/site-packages/google/cloud/happybase/table.py", line 428, in put
    batch.put(row, data)
  File "/Users/thebilly/cloud/python-docs-samples/.nox/py36-sample-bigtable-hello_happybase/lib/python3.6/site-packages/google/cloud/happybase/batch.py", line 275, in __exit__
    self.send()
  File "/Users/thebilly/cloud/python-docs-samples/.nox/py36-sample-bigtable-hello_happybase/lib/python3.6/site-packages/google/cloud/happybase/batch.py", line 111, in send
    table.mutate_rows(self._row_map.values())
  File "/Users/thebilly/cloud/python-docs-samples/.nox/py36-sample-bigtable-hello_happybase/lib/python3.6/site-packages/google/cloud/bigtable/table.py", line 508, in mutate_rows
    return retryable_mutate_rows(retry=retry)
  File "/Users/thebilly/cloud/python-docs-samples/.nox/py36-sample-bigtable-hello_happybase/lib/python3.6/site-packages/google/cloud/bigtable/table.py", line 684, in __call__
    mutate_rows()
  File "/Users/thebilly/cloud/python-docs-samples/.nox/py36-sample-bigtable-hello_happybase/lib/python3.6/site-packages/google/api_core/retry.py", line 270, in retry_wrapped_func
    on_error=on_error,
  File "/Users/thebilly/cloud/python-docs-samples/.nox/py36-sample-bigtable-hello_happybase/lib/python3.6/site-packages/google/api_core/retry.py", line 179, in retry_target
    return target()
  File "/Users/thebilly/cloud/python-docs-samples/.nox/py36-sample-bigtable-hello_happybase/lib/python3.6/site-packages/google/cloud/bigtable/table.py", line 718, in _do_mutate_retryable_rows
    retryable_rows.append(self.rows[index])
TypeError: 'dict_values' object does not support indexing

--- a/src/google/cloud/happybase/batch.py
+++ b/src/google/cloud/happybase/batch.py
@@ -108,7 +108,7 @@ class Batch(object):
     def send(self):
         """Send / commit the batch of mutations to the server."""
         table = self._table._low_level_table
-        table.mutate_rows(self._row_map.values())
+        table.mutate_rows(list(self._row_map.values()))
 
         self._row_map.clear()
         self._mutation_count = 0

